sdk/src/lib.rs

```diff
@@ -57,13 +57,16 @@ impl ProverClient {
     #[allow(clippy::new_without_default)]
     pub fn new() -> Self {
         dotenv::dotenv().ok();
-        let private_key = env::var("PRIVATE_KEY").unwrap_or_default();
-        if private_key.is_empty() {
-            Self { client: None }
-        } else {
+        let remote_proving = env::var("REMOTE_PROVING").unwrap_or_else(|_| String::from("false"));
+
+        if remote_proving == "true" {
```

Comment:
nit: Better practice to read this into a `bool`, but don't feel too strongly.

Reply:
updated